tests/Test/Dscp/Util/Serialise.hs

```diff
@@ -0,0 +1,17 @@
+module Test.Dscp.Util.Serialise
+    ( spec_Serialisation
+    ) where
+
+import Test.Common
+
+import Dscp.Util.Aeson (AsByteString, Base64Encoded, Versioned)
+import Test.Dscp.Serialise (aesonRoundtripProp)
+import Test.Dscp.Util.Instances ()
```

Comment:
@flyingleafe I did it like in your previous PR, but I'd prefer to have `Test.Dscp.Instances.*` modules instead - otherwise one has to remember each time where the required instance lies and whether is it present at all.